doc/source/geometric_manipulations.rst

```diff
@@ -0,0 +1,243 @@
+GeoPandas User Guide
+====================
+
+
+
+
+Set-theoretic Methods
+~~~~~~~~~~~~~~~~~~~~~
+
+**Attribute:** GeoSeries.boundary
```

Comment:
Is there a reason you didn't use the `.. attribute::` like it is in the current `user.rst` ?